test/Microsoft.OpenApi.CSharpAnnotations.DocumentGeneration.Tests/OpenApiDocumentGeneratorTests/OpenApiDocumentGeneratorTest.cs

```diff
@@ -1223,6 +1251,62 @@ public void NoOperationsToParseShouldReturnEmptyDocument(string openApiDocumentV
                 .BeEquivalentTo(openApiStringReader.Read(expectedDocument, out var _));
         }
 
+        [Theory]
+        [MemberData(nameof(GetTestCasesForPassingDescriptionShouldReturnCorrectDocument))]
+        public void PassingDescriptionShouldReturnCorrectDocument(
+            string testCaseName,
+            IList<string> inputXmlFiles,
+            IList<string> inputBinaryFiles,
+            string descriptionFile,
+            string expectedJsonFile)
+        {
+            _output.WriteLine(testCaseName);
+
+            var documents = new List<XDocument>();
+
+            documents.AddRange(inputXmlFiles.Select(XDocument.Load));
+
+            var input = new OpenApiGeneratorConfig(
+                documents,
+                inputBinaryFiles,
+                "1",
+                FilterSetVersion.V1)
+            {
+                OpenApiInfoDescription = File.ReadAllText(descriptionFile)
```

Comment:
> File.ReadAllText(descriptionFile) [](start = 40, length = 34)

nit, This test might fail if run on a UNIX platform since the newline in markdown might show up as just \n.

Suggest using ReadAllLines and join them with "\r\n" to match with your Json.

Reply:
That explains why it might be failing in app veyor, thanks will fix.

---
In reply to: [221389704](https://github.com/Microsoft/OpenAPI.NET.CSharpAnnotations/pull/158#discussion_r221389704) [](ancestors = 221389704)